src/r_main.c

```diff
@@ -367,10 +367,7 @@ static void R_InitTextureMapping(void)
       xtoviewangle[x] = (i<<ANGLETOFINESHIFT)-ANG90;
       // [FG] linear horizontal sky scrolling
       int angle = (0.5 - x / (double)viewwidth) * linearskyfactor;
-      if (angle >= 0)
-        linearskyangle[x] = angle;
-      else
-        linearskyangle[x] = ANG90 + angle;
+      linearskyangle[x] = angle >= 0 ? angle : ANGLE_MAX + angle;
```

Comment:
Could you put the condition in parentheses, please?

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/2160/commits/4c5caee4bf9fbb5b31569c6766570d182555e57d